yellowbrick/regressor/residuals.py

```diff
@@ -772,6 +830,12 @@ def residuals_plot(
         If set to 'density', the probability density function will be plotted.
         If set to True or 'frequency' then the frequency will be plotted.
 
+    qqplot : {True, False}, default: False
```

Comment:
Thanks for adding this here.  We do our best to keep our docstring up-to-date because this is how our docs are generated.